packages/edit-site/src/components/editor/index.js

```diff
@@ -135,6 +146,7 @@ function Editor( { settings: _settings } ) {
 								<Context.Provider value={ context }>
 									<FocusReturnProvider>
 										<KeyboardShortcuts.Register />
+										<Sidebar />
```

Comment:
Why is this sidebar out here now and how is it different from the new one below?

Reply:
Does this have the fills for the slot below?

Reply:
Why is that abstraction necessary?

Reply:
A name like `SidebarComplementaryAreaFills` would make it easier to grasp.